packages/@webex/plugin-meetings/src/webinar/index.ts

```diff
@@ -57,6 +50,27 @@ const Webinar = WebexPlugin.extend({
   updateCanManageWebcast(canManageWebcast) {
     this.set('canManageWebcast', canManageWebcast);
   },
+
+  /**
+   * Updates user roles and manages associated state transitions
+   * @param {object} payload
+   * @param {string[]} payload.oldRoles - Previous roles of the user
+   * @param {string[]} payload.newRoles - New roles of the user
+   * @returns {{isPromoted: boolean, isDemoted: boolean}} Role transition states
+   */
+  updateRoleChanged(payload) {
+    const isPromoted =
+      get(payload, 'oldRoles', []).includes(SELF_ROLES.ATTENDEE) &&
```

Comment:
suggestion: If you define newRoles and oldRoles then you don't need to do so many `get`s which might make this more readable

Reply:
Good suggestion, will change in the next PR